modules/core/src/main/scala/org/scalasteward/core/edit/EditAlg.scala

```diff
@@ -33,12 +34,14 @@ final class EditAlg[F[_]](
     fileAlg: FileAlg[F],
     logger: Logger[F],
     sbtAlg: SbtAlg[F],
+    scalafmtAlg: ScalafmtAlg[F],
     workspaceAlg: WorkspaceAlg[F],
     F: Sync[F]
 ) {
   def applyUpdate(repo: Repo, update: Update): F[Unit] =
     for {
       _ <- applyScalafixMigrations(repo, update)
+      _ <- scalafmtAlg.editScalafmtConf(repo)
```

Comment:
Wouldn't this update `.scalafmt.conf` for any `update`? I guess it should only be called if `update` is an `Update` for `scalafmt`, right?

Reply:
True.
Fixed in https://github.com/fthomas/scala-steward/pull/719/commits/10212f80ceccd522052d2690ec197b2d5f8f1dbb